src/Behat/Context.php

```diff
@@ -89,17 +120,12 @@ protected function assertNoException(): void
         }
     }
 
-    protected function disableDebounce(): void
-    {
-        $this->getSession()->executeScript('atk.options.set("debounceTimeout", 20)');
-    }
-
     /**
      * Sleep for a certain time in ms.
      *
-     * @Then I wait :arg1 ms
+     * @Then I sleep :arg1 ms
```

Comment:
keep Behat nomenclature using "Then I wait" - you can name the function as you want.

Reply:
I sleep and do nothing, sleep is explicit term for that.

Reply:
most of the Behat implementation I have seen use wait for Behat. I like to keep it that way.

Reply:
We can drop it completely :D

Reply:
No - because it can be useful when debugging new test.

Reply:
That is why I did not delete it :)